internal/storage/minio_chunk_manager.go

```diff
@@ -92,7 +93,7 @@ func newMinioChunkManagerWithConfig(ctx context.Context, c *config) (*MinioChunk
 		}
 		return nil
 	}
-	err = retry.Do(ctx, checkBucketFn, retry.Attempts(20))
+	err = retry.Do(ctx, checkBucketFn, retry.Attempts(2))
```

Comment:
Will reducing the number of retries cause stability problems?

Reply:
![screenshot-20220714-183533](https://user-images.githubusercontent.com/34002927/178964179-57c238ba-7bc4-4a88-9de9-ad2d5ebaed3c.png)
here it explain why the retry time is 2, and I just follow this instruction

Reply:
https://github.com/milvus-io/milvus/pull/18092/files#diff-8eecd044ceb67df3c20162d99ffe0dd511c5e932418942255d27a4b2034e6818L374-L378

Reply:
other components are also using miniochunkmanager, this configuration will also work for other components, I think this configuration can be changed in other pr.

Reply:
ok I will change back the retry times